desktop/release-notes.md

```diff
@@ -24,12 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.13.0
+2022-10-18
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added the new Dev Environments CLI `docker dev`.
+- Docker Desktop can now be installed to any drive and folder using the `--installation-dir`. Partially addresses [docker/roadmap#94](https://github.com/docker/roadmap/issues/94).
+- Two new security features have been introduced for Docker Business users, Settings Management and Enhanced Container Isolation. Read more about Docker Desktop’s new Hardened Desktop security model [here](/hardened-desktop/).
+
+### Upgrades
+
+- [Docker Scan v0.21.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.21.0)
+- [Go 1.19.2](https://github.com/golang/go/releases/tag/go1.19.2) to address [CVE-2022-2879](https://www.cve.org/CVERecord?id=CVE-2022-2879){: target="_blank" rel="noopener"}, [CVE-2022-2880](https://www.cve.org/CVERecord?id=CVE-2022-2880){: target="_blank" rel="noopener"} and  [CVE-2022-41715](https://www.cve.org/CVERecord?id= CVE-2022-41715){: target="_blank" rel="noopener"}
+- [Docker Engine v20.10.18](https://docs.docker.com/engine/release-notes/#201018), which contains a fix for [CVE-2022-36109](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-36109).
+- [Amazon ECR Credential Helper v0.7.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.7.0){: target="blank" rel="noopener" class=""}
+- [Docker Compose v2.11.2](https://github.com/docker/compose/releases/tag/v2.11.2)
+- [Kubernetes v1.25.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.25.2)
+- [Qemu 7.0.0](https://wiki.qemu.org/ChangeLog/7.0)
```

Comment:
Is this for CPU emulation (e.g. Intel on Apple Silicon) or for running a VM or both? IIRC there are 2 qemus (confusingly!)